### ISSUE TITLE
Mark items as not available if there's an upcoming reservation

### ISSUE THREAD

**Issue by maxverlaufen** [ArcoMul/leihbase#22]
An item that is reserved for the upcoming friday is still marked as "available now" on tuesday, for example, even though its technically not available. System should check if there is at least one "open" week (from friday to friday) before the next reservation or else mark the item accordingly.

**Comment by ArcoMul** [comment]
Yes that makes more sense, it's a bit complex to implement it like that though.

What do you think of not showing 'available now' at all, and instead of the red dot show a label like in this image:

![image](https://github.com/user-attachments/assets/b6f3afe4-286a-498c-addc-c09679aeee1c)

**Comment by ArcoMul** [comment]
Maybe, don't show 'Available now' badge
